docker/rumble/Dockerfile

```diff
@@ -16,10 +15,10 @@ RUN yum install -y cern-hadoop-config spark-bin-3.0 hadoop-bin-2.7.5 git git-cor
 ENV PATH $PATH:/usr/hdp/hadoop-2.7.5/bin:/usr/hdp/spark3/bin
 
 WORKDIR $WDIR
-RUN curl -k -L -O https://golang.org/dl/go1.16.4.linux-amd64.tar.gz && \
-    tar xfz go1.16.4.linux-amd64.tar.gz && \
+RUN curl -k -L -O https://golang.org/dl/go1.16.5.linux-amd64.tar.gz && \
+    tar xfz go1.16.5.linux-amd64.tar.gz && \
     mkdir $WDIR/gopath && \
```

Comment:
you don't need to setup gopath, instead create go.mod and go.sum in CMSMonitoring/src/go/Rumble which will keep all dependencies in place. Just visit this area and do:
```
go mod init github.com/dmwm/CMSMonitoring/src/go/Rumble
go mod tidy
```
Read about go.mod on internet!

Reply:
go.mod file is already in place under github.com/mrceyhun/CMSMonitoring/src/go/Rumble, PR is in review.